xapian-applications/omega/index_file.cc

```diff
@@ -545,6 +545,97 @@ index_add_document(const string & urlterm, time_t last_altered,
     }
 }
 
+void
+index_ctime_terms(Xapian::Document & doc, DirectoryIterator & d,
+		  const time_t & mtime, const time_t & ctime)
```

Comment:
`time_t` is just a signed integer type (at least on POSIX systems; if memory serves ISO C allows `double` but I don't think anything actually uses that).  There's no reason to pass it by const reference - just pass it by value instead.  We use `const reference` when copying is potentially expensive - e.g. `std::vector<std::string>` is O(n) in the number of entries to pass by value, but by const reference it's O(1).

Reply:
Yes, you are right. I am going to change it.